jenkins/ruby/distribution-build.jenkinsfile

```diff
@@ -0,0 +1,99 @@
+lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+
+pipeline {
+    options {
+        timeout(time: 1, unit: 'HOURS')
+    }
+    agent none
+    parameters {
+        string(
+            name: 'REPOSITORY',
+            description: 'Repository to release, e.g. https://github.com/opensearch-project/opensearch-ruby.git',
+            trim: true
+        )
+        string(
+            name: 'RELEASE_BRANCH',
+            description: 'Branch name to release, e.g. 1.2.0',
+            trim: true
+        )
+        string(
+            name: 'RUBY_INTERPRETER',
+            description: 'ruby, jruby, etc',
+            defaultValue: 'ruby',
```

Comment:
I think we should be defaulting and installing something more specific, e.g. `ruby-2.7.0`.